EIPIP Meeting 25 Notes.md

```diff
@@ -0,0 +1,318 @@
+
+**Micah** - Sure just for anybody who's not aware we have an auto merge bot. It goes up and down, has bugs, needs fixes like any other piece of code. Nick was the original author of it and he's very busy with other things and doesn't work on the eip group really anymore. So it would be nice if we had a new owner. Nick said you know this should really be moved to github actions. I agree Github actions are the perfect place for it. Github actions did not exist when Nick wrote it back in the day so it's a new tool that we have that just makes this a lot easier for group maintenance because anyone who can see the repo can see the action and tune it and play with it. It's obvious to anyone who uses github. So generally a good thing to do. Contracting is one of those tasks that is really easy to outsource because it doesn't require really much knowledge of ethereum. You just say hey we've got this tool here, tuned in python. It's very short. We need someone who can convert it into a github action so that's why i thought that if there's something to outsource this would be a good thing to do.
+
+**Hudson** - I think this is a good idea. I think that there's a bigger issue around funding that'll be a completely different topic in today's meeting, if we have time, or a different one. But I think this is an important enough change that we should ask the cat herders for funding for it now that their treasury has gotten bigger from the price increase, especially if we're going to be paying out an eth or something like that. But we'll have to talk to the rest of the cat herders about that so that should be on the agenda for their meeting and then if that is approved for funding from the cat herders we could do git coin. I'd prefer git coin but we don't have to use git coin for it necessarily.
+
+**Micah** - Today i learned the cat herders have money!
+
+**Hudson** - Cool 
+
+**Pooja** - Yeah but I believe getting an estimate of what amount of funding or the effort that we are looking for here would help us. 
+
+**Hudson**  - Yeah that would be helpful for a cat herders decision. Micah do you have any ideas or anybody else in the call?
+
+**Micah** - Eestimating software is the hardest job in the world. So relative to building an ethereum client this is a trivial task. This is one of those things like it depends on the skill level you hire and whether the person already has at least some knowledge of how this works. If you were to hire lightclient or me it would be an easier job than if you've hired some stranger but i think lightclient and me both are going to be more expensive and have little time available. If anyone wants to interrupt me please do. 
```

Comment:
```suggestion
**Micah** - Estimating software is the hardest job in the world. So relative to building an ethereum client this is a trivial task. This is one of those things like it depends on the skill level you hire and whether the person already has at least some knowledge of how this works. If you were to hire lightclient or me it would be an easier job than if you've hired some stranger but i think lightclient and me both are going to be more expensive and have little time available. If anyone wants to interrupt me please do. 
```